File: Npc/11004155.cs
using Maple2.Model.Enum;
using Maple2.Script.Npc;

/// <summary>
/// 11004155: Fufu
/// </summary>
public class _11004155 : NpcScript {
    protected override int First() {
        return 10;
    }

    // Select 0:
    // $script:0806025707010581$
    // - Ugh!
    protected override int Select() => 0;

    protected override int Execute(int selection) {
        switch (Id, Index++) {
            case (10, 0):
                // $script:0806025707010582$
                // - Ugh! I'm all stiff from standing around all day. I can't wait to get off work.
                return -1;
        }

        return default;
    }

    protected override NpcTalkButton Button() {
        return (Id, Index) switch {
            (10, 0) => NpcTalkButton.Close,
            _ => NpcTalkButton.None,
        };
    }
}

File: Npc/11004155.py
""" 11004155: Fufu """
from npc_api import Option, Script
import random


class Main(Script):
    def first(self) -> int:
        return 10

    def select(self) -> int:
        return 0

    def __0(self, pick: int) -> int:
        # $script:0806025707010581$
        # - Ugh!
        return None # TODO

    def __10(self, pick: int) -> int:
        # $script:0806025707010582$
        # - Ugh! I'm all stiff from standing around all day. I can't wait to get off work.
        return -1

    def button(self) -> Option:
        if (self.state, self.index) == (10, 0):
            return Option.CLOSE
        return Option.NONE
